README.md

```diff
@@ -2,272 +2,396 @@
-in outer scopes within an inner function. We'll also talk about what's happening
-under the hood when we run JavaScript code and how that impacts _identifier
-resolution_ and the _scope chain_.
-
-## Nested scopes and the scope chain
-
-In addition to the `#engineering` channel, every software engineer is a member
-of Flatbook's `#general` channel. Engineers can see all of the messages sent in
-both channels. If a message in `#general` piques our interest, we can refer to
-the message in `#engineering` despite the fact that it was posted in `#general`.
-To bend the analogy back towards functions and scope, everything _declared_ in
-`#general` is accessible in `#engineering`. `#general`, our global scope, is
-effectively the _outer scope_ for `#engineering`.
-
-For a function declared at the top level of our JavaScript file (that is, not
-declared inside of another function), its outer scope is the _global scope_.
-When that new function is invoked, it can access all of the variables and
-functions declared in the global scope. Upon invocation, the function creates a
-new scope and **retains a reference to the _outer scope_ in which it was
-declared**. Inside the new function's body, in addition to variables and
-functions declared in that function, **we also have access to variables and
-functions declared in the outer scope**. Let's see that in action:
+Every variable, every function defined in JavaScript belongs to a scope -
+whether it's global, functional, or block. Previously, we've learned how to
+deduct what scope a variable belongs to. As programmers, when we read or write
+code, we can tell what scope a variable belongs to by just looking at its
+surroundings. If it's inside a function, it's scoped to that function, and so
+on.
 
-```js
-const globalVar = 1;
+Subsequently, we can tell what parts of our code have access to any given
+variable. If it's inside a function, only anything _inside_ that function has
+access to it. If it's at the top level of a file, it's globally scoped,
+therefore any part of our code, after the variable has been defined, has access
+to it.
+
+Again, we know this by just looking at everything around it. So, how does a
+computer do it? When a JavaScript file runs, it has access to something called
+the **scope chain**. This chain provides all the scopes currently accessible by
+a given section of code, such as a function, or an `if` statement.
+
+In this lesson, we'll discuss how the scope chain allows us to access variables
+and functions declared in outer scopes within an inner function.
+
+## Nested Scopes
+
+Before we further examine the scope chain, there's an elephant in the room we
+must discuss. **Scopes can be nested**.
+
+This should not sound surprising. After all, we have already seen examples of
+nested scopes. You've even written code with nested scopes. The levels of scopes
+themselves demonstrate how they can be nested.
```

Comment:
This wording is a little vague "The levels of scopes themselves demonstrate how they can be nested". Is this referring to global scope, function scope, and block scope? If so, we could change this to something like "The fact that we can have function and block scope within our global scope already demonstrates how scopes can be nested.